spec/controllers/content_blocks_controller_spec.rb

```diff
@@ -81,12 +83,17 @@
           FactoryGirl.create(:content_block, name: 'featured_researcher')
         end
 
+        let!(:about) do
```

Comment:
Yes, but it doesn't conform to the new patterns: https://github.com/backus/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/let_setup.rb